src/style/app.module.css

```diff
@@ -639,17 +683,52 @@
 
 .regularBtn:is(:hover, :active) {
   background-color: #286fe0 !important;
+  color: var(--dark-grey-font);
+  border-color: var(--search-button-border) !important;
+}
+
+.regularBtn:is(:hover, :active) {
+  background-color: #286fe0 !important;
+  color: var(--dark-grey-font);
   border-color: var(--search-button-border) !important;
 }
 
+.activeTab {
+  background-color: var(--grey-light);
+  color: var(--grey-text);
+  border-color: var(--grey-text);
+  align-items: center;
+}
+
+.activeTab:is(:hover, :focus, :active) {
+  background-color: var(--grey-dark) !important;
+  color: var(--white);
+  border-color: var(--grey-dark) !important;
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`